lib/aggregate/base.rb

```diff
@@ -28,6 +28,14 @@ def <=>(other)
       end || 0
     end
 
+    def self.hashed_keys
```

Comment:
This probably isn't needed. Just assume the configuration is a list. Though we should probably have the secret keys base64 encoded. So maybe a method like this one should be used to decode them..